openmrs/apps/clinical/app.json

```diff
@@ -27,6 +27,11 @@
         "enableRadiologyOrderOptions":["Urgent","NeedsPrint"],
         "enableLabOrderOptions":["Urgent", "NeedsPrint"],
         "quickPrints":false,
+        "ipdButton": {
```

Comment:
As we have discussed, here also the name should change to adtNavigationConfig

Reply:
No. For config we are not making any changes right? @pramidatumma  @SwathiVarkala